SUMO-251067 | Apoorv | Add. MessagingAppsV2Migration (#1)
Apoorvkudesia-sumologic/sumologic-documentation: docs/integrations/containers-orchestration/activemq.md

Configuring log and metric collection for the ActiveMQ App includes the following tasks:

### Step 1: Fields in Sumo Logic

> Following fields will be created as part of app installation process, For information on setting up fields, see [Sumo Logic Fields](https://help.sumologic.com/docs/manage/fields/)

Reviewer: ```suggestion
Following [fields]((https://help.sumologic.com/docs/manage/fields/)) will be created as part of app installation process.
```

<TabItem value="k8s">

> If you're using ActiveMQ in a Kubernetes environment, then these fields will be created:

Reviewer: We need to get this out of tab view. And give a clear idea that fields created in non-K8S will be created always and pod_label_* fields will get created when we are dealing with K8S

    ```
   5. Sumo Logic Kubernetes collection will automatically start collecting logs from the pods having the annotations defined above.

> 3. **FERs to normalize the fields in Kubernetes environments**. Labels created in Kubernetes environments automatically are prefixed with `pod_labels`. To normalize these for our app to work, we will have a Field Extraction Rule automatically created for Messaging Application Components named as **App Observability - Messaging**

Reviewer: ```suggestion
3. **FERs to normalize the fields in Kubernetes environments**. Labels created in Kubernetes environments automatically are prefixed with `pod_labels`. To normalize these for our app to work, we will have a Field Extraction Rule automatically created with named as **App_Observability_ActiveMQ_messaging**
```

Reviewer: Make the FER name same to that which is declared in tf file.

</Tabs>

> ## ActiveMQ Monitors

Reviewer: ```suggestion
## Installing the ActiveMQ app

import AppInstall2 from '../../reuse/apps/app-install-v2.md';

<AppInstall2/>

## Viewing ActiveMQ dashboards​

import ViewDashboards from '../../reuse/apps/view-dashboards.md';

<ViewDashboards/>

## ActiveMQ Monitors
```
